apps/frontend/src/pages/projects/components/project-create-form/project-create-form.tsx

```diff
@@ -1,29 +1,39 @@
 import { Button, Input } from "~/libs/components/components.js";
-import { useAppForm, useCallback } from "~/libs/hooks/hooks.js";
+import { useAppForm, useCallback, useState } from "~/libs/hooks/hooks.js";
 import {
 	type ProjectCreateRequestDto,
 	projectCreateValidationSchema,
 } from "~/modules/projects/projects.js";
 
 import { DEFAULT_PROJECT_CREATE_PAYLOAD } from "./libs/constants/constants.js";
+import {
+	ProjectValidationMessage,
+	ProjectValidationRule,
+} from "./libs/enums/enums.js";
 import styles from "./styles.module.css";
 
 type Properties = {
 	onSubmit: (payload: ProjectCreateRequestDto) => void;
 };
 
 const ProjectCreateForm = ({ onSubmit }: Properties): JSX.Element => {
-	const { control, errors, handleSubmit } = useAppForm<ProjectCreateRequestDto>(
-		{
+	const { control, errors, handleSubmit, watch } =
+		useAppForm<ProjectCreateRequestDto>({
 			defaultValues: DEFAULT_PROJECT_CREATE_PAYLOAD,
 			validationSchema: projectCreateValidationSchema,
-		},
-	);
+		});
+
+	const [isSubmitted, setIsSubmitted] = useState<boolean>(false);
+
+	const descriptionValue = watch("description", "");
+	const charCount = descriptionValue.length;
```

Comment:
it would be better to rename charCount to description length, but there is no point in this because we use it only once, so let's remove it
```suggestion
	const description = watch("description", "");
```

Reply:
This variable is used to display the length of the description text entered. I can remove it and then use descriptionValue.length. Is it better?

Reply:
if it's used in one place it will be ok